File: CinemaProject.Angular/src/app/AdminPages/Models/typeOfSeat.ts
class TypeOfSeat {
  constructor(
    public id: string,
    public cinemaId: string,
    public name: string,
    public extraPaymentPercent: number,
    public color: string
  ) {}
}

class TypeOfSeatView {
  constructor(
    public id: string,
    public cinemaName: string,
    public name: string,
    public extraPaymentPercent: string,
    public color: string
  ) {}
}

export { TypeOfSeat, TypeOfSeatView };

File: CinemaProject.WebAPI/CinemaProject.BLL/Models/TypeOfSeat.cs
using System;

namespace CinemaProject.BLL.Models
{
    public class TypeOfSeat
    {
        public Guid Id { get; set; }
        public Guid CinemaId { get; set; }
        public string Name { get; set; }
        public int ExtraPaymentPercent { get; set; }
    }
}
